rules/macos/credential_access_suspicious_web_browser_sensitive_file_access.toml

```diff
@@ -3,20 +3,20 @@ creation_date = "2020/01/04"
 integration = ["endpoint"]
 maturity = "production"
 min_stack_comments = "New fields added: required_fields, related_integrations, setup"
-min_stack_version = "8.3.0"
-updated_date = "2023/11/15"
+min_stack_version = "8.11.0"
```

Comment:
Any specific reason for 8.11.0? We should note it in the comments.

Reply:
8.11 is when the file open data source was added.